cc-db-service/migrations/20221018130025-create-api-configuration.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable no-undef */
```

Comment:
Does this eslint line have to be here since migrations -folder is mentioned in the .eslintignore -file? The same goes for the other files in the migrations and seeders -folders.

Reply:
yes, atleast my code editor did not register that the folders were listed in the ignore file :D